### ISSUE TITLE
BUG: positionFixed applies `position:block`

### ISSUE THREAD

**Issue by Roman991** [tonysamperi/ngx-float-ui#3]
**Describe the bug**
[positionFixed]="true" not applies `position:fixed` to  float-ui-container element

**To Reproduce**
``` html
                    <span
                      [floatUi]="floatUiTemplate"
                      [showDelay]="600"
                      [positionFixed]="true"
                      [showTrigger]="triggerHover"
                      applyClass="w-80"
                  ></span>
                  <float-ui-content #floatUiTemplate>
                      <span class="w-full">
                         floating element
                      </span>
                  </float-ui-content>
```

> not working float-ui-container from ngx-float-ui
```
element.style {
    display: block;
    opacity: 0;
    left: -193.3px;
    top: -153.071px;
}
```

the same ngxp__container from ngx-popperjs
```
...
display: fixed;
...
```

**Comment by tonysamperi** [comment]
Fixed in v 17.0.1 thanks for reporting! 🎉

**Comment by tonysamperi** [comment]
By the way `display: fixed` sounds unlikely 😂😂😂
